docs/docs/videos.md

```diff
@@ -168,6 +168,6 @@ Walk-through by [Andres Suarez](https://github.com/zertosh) on how [SoundCloud](
 
 ### Introducing React Native (+Playlist) - React.js Conf 2015
 
-<iframe width="650" height="366" src="https://www.youtube.com/embed/KVZ-P-ZI6W4?list=PLb0IAmt7-GS1cbw4qonlQztYV1TAW0sCr" frameborder="0" allowfullscreen></iframe>
+<iframe width="650" height="366" src="https://www.youtube-nocookie.com/embed/KVZ-P-ZI6W4?list=PLb0IAmt7-GS1cbw4qonlQztYV1TAW0sCr" frameborder="0" allowfullscreen></iframe>
```

Comment:
I'm fine with this change but we're not all going to remember to do this for all future posts so in all honesty it's probably a futile effort. The fact that https://www.youtube-nocookie.com/ is a (blank) 404 page. The [Google results](ahttps://www.google.com/search?q=youtube-nocookie&ie=utf-8&oe=utf-8) are laughable and none of them are Google mentioning their support for it.